CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Updated deserialization of `bitmap` for custom CPU templates to allow usage
   of '_' as a separator.
 - Changed the strip feature of `cpu-template-helper` tool to operate bitwise.
+- Better logs during validation of cpu id in snapshot restoration path.
```

Comment:
I think we can be more specific here: let's say that we don't fail anymore if host and snapshot vendor IDs don't match.

Reply:
It never failed if ids were not the same (it was printing error, but that's it). It only failed if it couldn't get host or snap ids.

Reply:
yeah, sorry, I meant that. I think we should mention that we don't fail in that case now.

Reply:
done